application/common/models/User.php

```diff
@@ -1183,6 +1183,29 @@ public static function deleteInactiveUsers()
         ]);
     }
 
+
+    /**
+     * Delete all user records that are inactive and have not been updated recently.
+     */
+    public static function getInactiveUsers()
```

Comment:
Looks like you could (should) use `getInactiveUsers` in `deleteInactiveUsers` assuming the difference is insignificant. It's confusing to me without greater analysis why the two queries are different (`last_changed_utc` vs `last_login_utc` + `created_utc`). If they ARE different, it would be less confusing to rename this function (and possibly the other as well) to describe why they are different.

Reply:
After talking with Phillip, methods should be different, and I renamed them.